tests/platform_tests/daemon/test_ledd.py

```diff
@@ -180,16 +149,17 @@ def test_pmon_ledd_kill_and_start_status(check_daemon_status, duthosts, rand_one
     duthost.stop_pmon_daemon(daemon_name, SIG_KILL, pre_daemon_pid)
 
     daemon_status, daemon_pid = duthost.get_pmon_daemon_status(daemon_name)
-    logger.info("{} daemon got killed unexpectedly and it is {} with pid {}".format(daemon_name, daemon_status, daemon_pid))
+    logger.info("{} daemon got killed unexpectedly and it is in {} with pid {}".format(daemon_name, daemon_status, daemon_pid))
```

Comment:
check the daemon_status before logging "daemon got killed"

Reply:
The daemon_status check is followed right after but this is just added for debugging purpose. I can remove this part.

Reply:
better put a log "sending SIG_KILL to daemon {}" before calling stop_pmon_daemon() for debugging and remove this

Reply:
The test name(test_pmon_ledd_kill_and_start_status) is already indicating this is sending SIG_KILL command.